CHANGELOG.md

```diff
@@ -19,6 +19,8 @@
   ([#6930](https://github.com/mitmproxy/mitmproxy/pull/6930), @errorxyz)
 * Parse compressed domain names in ResourceRecord data
   ([#6954](https://github.com/mitmproxy/mitmproxy/pull/6954), @errorxyz)
+* Fix a bug where mitmweb's flow list would not stay at the bottom.
+  ([#7008](https://github.com/mitmproxy/mitmproxy/pull/7008), @mhils)
```

Comment:
#7008 is this PR

![harold](https://github.com/mitmproxy/mitmproxy/assets/679144/92a186e5-0fa7-45c0-ab1f-36e8820fe101)

Reply:
Yes, this was fixed in this PR. 😛